pkgs/desktops/pantheon/default.nix

```
@@ -121,6 +121,8 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   granite = callPackage ./granite { };
```

can we add `granite6` as package, so it can be used specifically?
```suggestion
  granite6 = callPackage ./granite { };
  granite = granite6;
```

---

Not sure - the number "7" is hardcoded [everywhere](https://github.com/elementary/granite/search?q=granite-7) on upstream so I will probably just remove `pantheon.granite` when it is removed from [PPA](https://launchpad.net/~elementary-os/+archive/ubuntu/stable?field.series_filter=jammy) and just keep `pantheon.granite7` :thinking: